frameworks/PHP/symfony/composer.json

```diff
@@ -0,0 +1,72 @@
+{
+	"name" : "symfony/framework-standard-edition",
+	"license" : "MIT",
+	"type" : "project",
+	"description" : "The \"Symfony Standard Edition\" distribution",
+	"autoload" : {
+		"psr-4" : {
+			"" : "src/"
+		},
+		"classmap" : [
+			"app/AppKernel.php",
+			"app/AppCache.php"
+		]
+	},
+	"autoload-dev" : {
+		"psr-4" : {
+			"Tests\\" : "tests/"
+		},
+		"files" : [
+			"vendor/symfony/symfony/src/Symfony/Component/VarDumper/Resources/functions/dump.php"
+		]
+	},
+	"require" : {
+		"php" : ">=5.5.9",
+		"doctrine/doctrine-bundle" : "^1.6",
+		"doctrine/orm" : "^2.5",
+		"incenteev/composer-parameter-handler" : "^2.0",
+		"sensio/distribution-bundle" : "^5.0.19",
+		"sensio/framework-extra-bundle" : "^3.0.2",
+		"symfony/monolog-bundle" : "^3.1.0",
+		"symfony/polyfill-apcu" : "^1.0",
+		"symfony/swiftmailer-bundle" : "^2.3.10",
+		"symfony/symfony" : "^3.3",
```

Comment:
would be great to replace y standalone components, the default config adds overhead otherwise

Reply:
We should really use Flex instead.